inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -808,16 +808,26 @@ def affiliations_for_hidden_collections(obj):
     return [affiliations_mapping[affiliation] for affiliation in affiliations_set]
 
 
+def reports_for_hidden_collections(obj):
+    hidden_collection_mapping = current_app.config.get("AFFILIATIONS_TO_HIDDEN_COLLECTIONS_MAPPING", {})
+    hidden_collections = set()
+    for report in obj.data.get('report_number', []):
+        hidden_collections.update([match.upper() for match in re.findall('cern|fermilab', report['value'], re.IGNORECASE)])
```

Comment:
it would be more readable if we split this into more lines

Reply:
also we could directly update the set with the `hidden_collection_mapping` value, instead of creating a new set

Reply:
But `hidden_collection_mapping` is a different thing, there are also other values. I would prefer to store matched collections separately